CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -585,10 +585,6 @@ public function parseActionSchedule($values) {
     if ($this->_action & CRM_Core_Action::UPDATE) {
       $params['id'] = $this->_id;
     }
-    elseif ($this->_action & CRM_Core_Action::ADD) {
-      // we do this only once, so name never changes
-      $params['name'] = CRM_Utils_String::munge($params['title'], '_', 64);
-    }
```

Comment:
@totten this was the offending form code, now redundant with the DAO which handles it better.